apiserver/facades/client/subnets/cache.go

```diff
@@ -22,12 +21,10 @@ import (
 	providercommon "github.com/juju/juju/provider/common"
 )
 
-var logger = loggo.GetLogger("juju.apiserver.common.networkingcommon")
-
 // addSubnetsCache holds cached lists of spaces, zones, and subnets, used for
 // fast lookups while adding subnets.
 type addSubnetsCache struct {
```

Comment:
Are we going to ditch this too at some point?

Reply:
Yes.  Trying to make the changes in stages.  First is to refactor, second to get rid of the cache, last is to change to mock tests.